kernel/src/engine/parquet_stats_skipping.rs

```diff
@@ -73,23 +141,28 @@ pub(crate) trait ParquetStatsSkippingFilter {
     /// If the result was FALSE, it forces both inner and outer AND to FALSE, as desired. If the
     /// result was TRUE or NULL, then it does not contribute to data skipping but also does not
     /// block it if other legs of the AND evaluate to FALSE.
-    fn apply_sql_where(&self, filter: &Expression) -> Option<bool> {
-        use Expression::*;
-        use VariadicOperator::And;
+    // TODO: If these are generally useful, we may want to move them into PredicateEvaluator?
```

Comment:
yea agree maybe punt on this for now and create a follow up issue so that if we find other instances of usage then we know we should just lift this up in `PredicateEvaluator`?